src/kms.ts

```diff
@@ -1190,10 +1208,10 @@ export class KmsClient {
       WrappingMethod.Encrypt,
       new EncryptionKeyInformation(encryptionKeyUniqueIdentifier),
     )
-    const object = await this.getObject(
-      uniqueIdentifier,
+    const object = await this.getObject(uniqueIdentifier, {
+      unwrap: true,
```

Comment:
is it OK/logic to set unwrap to true here, in a function named getWrappedKey ? If so, maybe rename this function ?

Reply:
true, removing it, thank you